example-deck.gl.md

```diff
@@ -0,0 +1,400 @@
+Plotting CO-Values using deck.gl
+================================
+
+In this example we explain how to use [deck.gl](https://deck.gl/#/),
+a WebGL-powered framework for data visualization created by [Uber](https://www.uber.com),
```

Comment:
```suggestion
a WebGL-powered framework for data visualization created by [Uber](https://uber.com),
```

Reply:
@lkiesow  https://uber.com is redirected to https://www.uber.com. Would you still use https://uber.com?

```bash
curl -v https://uber.com
...
< HTTP/2 301 
< location: https://www.uber.com/
...
```

Reply:
No strong feelings but I tend to prefer using the main domain name instead of a subdomain whenever possible as long as it gets me where I want